src/assigner.py

```diff
@@ -0,0 +1,46 @@
+from settings import ID_DICT_LIST_ITEM, ASSIGN_TO_TEST_PLANS, ASSIGN_TO_FEATURE, TP_URL, ACCESS_TOKEN
+from copy import deepcopy
+import requests
+
+class Assigner():
```

Comment:
Minor thing in python3 you can just do this:
```suggestion
class Assigner:
```